components/var_dumper/advanced.rst

```diff
@@ -97,7 +97,12 @@ Dumpers
 
 A dumper is responsible for outputting a string representation of a PHP variable,
 using a :class:`Symfony\\Component\\VarDumper\\Cloner\\Data` object as input.
-The destination and the formatting of this output vary with dumpers.
+The destination and the formatting of this output vary with dumpers and are 
+influenced by two environment variables: 
+If ``DUMP_STRING_LENGTH`` is set, then
+the length of a string is displayed next to its content. 
+If ``DUMP_LIGHT_ARRAY`` is set, 
+then arrays are not displayed completely.
```

Comment:
Would it make sense to render the two sentences as a list? This would then look like this:

```rst
The destination and the formatting of this output vary with dumpers and are
influenced by two environment variables:

* If ``DUMP_STRING_LENGTH`` is set, then the length of a string is displayed
  next to its content.
* If ``DUMP_LIGHT_ARRAY`` is set, then arrays are not displayed completely.
```